blackjax/mcmc/random_walk.py

```diff
@@ -0,0 +1,58 @@
+from typing import Callable, Tuple
+
+import jax
+from jax import numpy as jnp
+
+from blackjax.mcmc.rmh import RMHInfo, RMHState, rmh
+from blackjax.types import Array, PRNGKey, PyTree
+from blackjax.util import generate_gaussian_noise
+
+__all__ = ["kernel", "normal"]
+
+
+def kernel(random_step):
```

Comment:
Everything from `rmh.py` should be moved here (see details below) but in the more general implementation of random walk metropolis hastings (not only gaussian RWMH).

Ideally, we'd use the functions from `proposal.py` and objects from `integrators.IntegratorState` to do what `rmh.py` currently does.